app/models/Campaign.js

```diff
@@ -34,8 +34,12 @@ const campaignSchema = new mongoose.Schema({
   msg_menu_signedup_external: String,
   msg_menu_signedup_gambit: String,
   msg_no_photo_sent: String,
-  msg_relative_reminder_signup: String,
-  msg_relative_reminder_reportback: String,
+
+  // Exposed messages.
+  messages: {
```

Comment:
I think that's reversed @sergii-tkachenko -- but I follow. It's semantics but Signup Reminder message sounds ;ike "Hey you haven't signed up for this yet" -- i don't think it should be called a reminder message.